packages/web-components/fast-foundation/src/accordion-item/accordion-item.template.ts

```diff
@@ -27,24 +27,21 @@ export function accordionItemTemplate<T extends FASTAccordionItem>(
                 id="${x => x.id}"
                 @click="${(x, c) => x.clickHandler(c.event as MouseEvent)}"
             >
+                ${startSlotTemplate(options)}
                 <span class="heading-content" part="heading-content">
                     <slot name="heading"></slot>
                 </span>
+                ${endSlotTemplate(options)}
+                <span class="expand-collapse-icon" part="expand-collapse-icon" aria-hidden="true">
+                    <slot name="expand-collapse-icon">
+                        ${staticallyCompose(options.expandCollapseIcon)}
+                    </slot>
+                <span>
```

Comment:
Revisiting this and I don't think we want these within the button itself because it will group all content as part of the label. While I can definitely see a case for the `expand-collapse-icon`, moving the others in could actually generate some new a11y issues for folks under the assumption that the only slotted content here is icons (which I don't think would be an accurate assumption).

Reply:
I tested this on Anchor, for instance with a Badge, and it reads the badge content the same in both cases. In some cases that may make sense, like announcing something as "New". If they are Buttons instead, focus goes to the accordion button, then tab to start, end, then the next accordion item, which seems like the flow I expect.

Reply:
Note that this change (at least currently) is _only_ the sequence of items, not the hierarchy. This should not have any a11y impact that I know of, it only seeks to align the named intent of the "start" and "end" slots with their document order to minimize the need to resequence with css.

Reply:
I think the intent with not including these as part of the button was that there may be scenarios where people may place _other_ interactive elements within these slots and placing those inside the button would create a problem there. I think this is a hierarchy change because these are moving to be nested within the button, which means anything projected into the slot is a child of the button now instead.

Reply:
Uh, looks like I didn't look at the diff correctly. Of course those elements moved.

I looked back at the spec, and I don't see a way to make both of the following statements true given the additional elements we have. I agree that the previous hierarchy (I've shifted the sequence) is best.

> The button element is the only element inside the heading element.
> That is, if there are other visually persistent elements, they are not included inside the heading element.